code/modules/medical/genetics/bioHolder.dm

```diff
@@ -16,17 +16,21 @@ var/list/datum/bioEffect/mutini_effects = list()
 
 /datum/appearanceHolder
 	//Holds all the appearance information.
+	var/customization_first_color_carry = "#101010"
```

Comment:
some notation what these vars mean, please

Reply:
This good?